test/test.js

```diff
@@ -348,15 +368,9 @@ describe( 'rollup', function () {
 				( config.skip ? it.skip : config.solo ? it.only : it )( dir, function ( done ) {
 					process.chdir( path.resolve( CLI, dir ) );
 
-					if (os.platform() === 'win32') {
-						config.command = "node " + path.resolve( __dirname, '../bin' ) + path.sep + config.command;
-					}
+					const command = 'node ' + path.resolve( __dirname, '../bin' ) + path.sep + config.command;
```

Comment:
`path.join('node ', path.resolve( __dirname, '../bin' ), config.command)`

@Rich-Harris @TrySound Functional programming FTW. 

Net/Net same result at the end of the day.